packages/reports/src/Components/Routes/binary-link.tsx

```diff
@@ -1,15 +1,16 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import { PlatformContext } from '@deriv/shared';
 import getRoutesConfig from 'Constants/routes-config';
 import { findRouteByPath, normalizePath } from './helpers';
 
-const BinaryLink = ({ active_class, to, children, ...props }) => {
-    const { is_appstore } = React.useContext(PlatformContext);
+type TBinaryLink = {
+    active_class: string;
+    to: string;
+};
 
-    const path = normalizePath(to);
-    const route = findRouteByPath(path, getRoutesConfig({ is_appstore }));
+const BinaryLink = ({ active_class, to, children, ...props }: React.PropsWithChildren<Partial<TBinaryLink>>) => {
+    const path = normalizePath(to!);
```

Comment:
```suggestion
    const path = normalizePath(to);
```

Is this `!` extra?

Reply:
@niloofar-deriv it has been added due to prevent this TS error
`Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.ts(2345)`

Reply:
```suggestion
    const path = normalizePath(to || '');
```

What do u think about this? :thinking: